docs/sources/tempo/traceql/_index.md

```diff
@@ -302,3 +302,9 @@ Find any trace where any span has an `http.method` attribute set to `GET` as wel
 ```
 { span.http.method = "GET" && status = ok } && { span.http.method = "DELETE" && status != ok }
 ```
+
+Find any trace with a `deployment.environment` attribute that matches the regex `prod-.*` and `http.status_code` attribute set to `200`:
+
+```
+{ .deployment.environment = "prod-.*" && .http.status_code = 200 }
```

Comment:
The regex operator needs to be used to do the comparison correctly. I'd also prefer to keep as many example as possible to use the `resource` and `span` scopes where appropriate. it's so much more efficient!

```suggestion
{ resource.deployment.environment =~ "prod-.*" && span.http.status_code = 200 }
```

Reply:
Oh! The example of the regex operator was the only intention of this PR. I think I copied things around and missed updating the example :palmface: sorry about that, and thanks for the fix